docs/references/filters.md

```diff
@@ -62,9 +62,20 @@ class ColorFilter:
     fruits: FruitFilter
 ```
 
-## Overriding default filtering method
+## Custom filters and overriding default filtering method
 
-TODO
+You can define custom filter methods and override default filter methods by defining your own resolver.
+
+```python
+@strawberry.django.filters.filter(models.Fruit)
+class FruitFilter:
+    is_banana: bool
```

Comment:
A better example would be to type this as `bool | None`, to show that we can have non-mandatory filter attributes.

(or define a mandatory and a non-mandatory one)